src/main/java/com/palonskiy/serice/BookService.java

```diff
@@ -3,32 +3,23 @@
 import com.palonskiy.dto.AuthorDto;
 import com.palonskiy.dto.BookAuthorDto;
 import com.palonskiy.dto.BookDto;
-import com.palonskiy.model.Book;
 
 import java.util.List;
 
 public interface BookService {
-    List<BookDto> get();
+    List<BookDto> getAll();
 
     List<AuthorDto> getBookAuthors(int id);
 
-    void addOnlyBook(BookDto bookDto);
-
-    Long getId();
-
     void add(BookAuthorDto bookAuthorDto);
 
     void delete(int id);
 
-    Book checkIfExist(String name);
+    Boolean checkIfExist(String name);
```

Comment:
use primitive where possible

Reply:
Have changed to primitive types everywhere except crudDao.getById. Add test fails with primitive type. Couldn't find the reason